doc/_sphinx/theme/doctools.js

```diff
@@ -86,11 +93,11 @@ const _highlightText = (thisNode, text, className, index) => {
 /**
  * Small JavaScript module for the documentation.
  */
-const Documentation = {
+const Documentation_flames = {
```

Comment:
How come it needed a rename? And shouldn't this be singular if the rename is needed?

Reply:
It also looks weird with a starting capital letter and then snake-case?

Reply:
Same as above.  It is the name of the custom theme the docs uses, so it stays consistent.

Reply:
But shouldn't it be CamelCase?

Reply:
The theme name is "flames".  I was keeping everything consistent.  If you want it all changed, I can do that, it just seems inconsistent.

Reply:
I understood that, what I meant is, shouldn't the variable name be `DescriptionFlames` and not with underscore?

Reply:
What I'm saying is,  "Documentationflames" was worse than "Documentation_flames" which is more consistent than "DocumentationFlames".  I don't care and it has no impact. If you want the latter, fine by me.

Reply:
Updated and pushed.